avalara_salestax/models/avalara_salestax.py

```diff
@@ -118,7 +118,7 @@ def _get_default_company(self):
         default=True, help="Uncheck the active field to hide the record")
     company_id = fields.Many2one(
         'res.company', 'Company', required=True,
-        default='_get_default_company',
+        default=_get_default_company,
```

Comment:
Unrelated fix. Noticed default company method was not called because evaluation of quoted methods is only supported for compute methods, not for default methods.